### ISSUE TITLE
Can't Attach Any File

### ISSUE THREAD

**Issue by badarulaa** [whatsplay/whatsapp-play#426]
Hello.

When i try running chat using terminal, and run command #_FILE, why always appear this problem

No node found for selector: #main > header > div > div > div > span > div > div > ul > li:nth-child(3) > button
  File "/home/badar/GitHub/whatsapp-play/wplay/utils/io.py", line 52, in send_file
    await page.click(whatsapp_selectors_dict['choose_file'])
  File "/home/badar/GitHub/whatsapp-play/wplay/terminal_chat.py", line 86, in chat
    await io.send_file(page)
  File "/home/badar/GitHub/whatsapp-play/wplay/__main__.py", line 165, in get_and_match_args
    await terminal_chat.chat(args.target)
  File "/home/badar/GitHub/whatsapp-play/wplay/__main__.py", line 209, in main
    await get_and_match_args(parser)
  File "/home/badar/GitHub/whatsapp-play/wplay/__main__.py", line 216, in <module>
    asyncio.get_event_loop().run_until_complete(main())

thank you for advance

**Comment by rpotter12** [comment]
Hi @badarulaa, this error is caused because whatsapp changes their whatsapp-web HTML format after every few months. So the selector used in this software is older one. To fix this error we need to update the selectors.

**Comment by badarulaa** [comment]
> Hi @badarulaa, this error is caused because whatsapp changes their whatsapp-web HTML format after every few months. So the selector used in this software is older one. To fix this error we need to update the selectors.

Well i see, so we need to search which one a new selector, right ?

**Comment by rpotter12** [comment]
Yes, we need to inspect(ctrl + shift + i) on whatsapp-web and update the selectors in https://github.com/whatsplay/whatsapp-play/blob/master/wplay/utils/helpers.py

**Comment by gurpreet24112004** [comment]
Where can we attach the file of numbers
